src/threadpool.c

```diff
@@ -37,16 +37,18 @@ static void uv__req_init(uv_loop_t* loop,
     uv__req_init((loop), (uv_req_t*)(req), (type))
 #endif
 
+#include <limits.h>
 #include <stdlib.h>
 
 #define MAX_THREADPOOL_SIZE 128
+#define IDLE_THREAD_TIMEOUT 5e9  /* 5 seconds in nanoseconds. */
```

Comment:
Arbitrary timeout.  Maybe this should be configurable with an option for 'never'.